Header header
C2_BHV m_bhv
float32 lat
float32 lon
float32 heading
float32 mission_pt_radius